scheduler/src/cook/components.clj

```diff
@@ -61,15 +62,31 @@
                 [:headers "Cache-control"]
                 "max-age=0"))))
 
+(def rate-limiters
+  {:create-job-rate-limiter
+   (fnk [settings]
```

Comment:
Can this become `[:settings rate-limit]`? Then the plumbing library also validates that the `rate-limit` key exists in the settings object.

Reply:
QQ: I'm unfamiliar enough with the plumbing library to not quite understand what this is asking for. Help please. :)

Reply:
`[:settings rate-limit]` is using destructruing support in the settings entry and also validating that a key named `:rate-limit` exists.